plugins/upload/guest-js/index.ts

```diff
@@ -60,6 +61,7 @@ async function download(
     filePath,
     headers: headers ?? {},
     onProgress,
+    body: body ?? "",
```

Comment:
I think it's fine to just send null and use `Option<&str>` in Rust. Feels a bit cleaner but i don't really care that much.

Reply:
I tried with Option<&str> in Rust but it gives me problems on condition